### PR TITLE
edit to constraints page

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: docs/docs/constraints.md [hunk 1]
```diff
@@ -4,31 +4,31 @@ title: Constraints
 
 # Constraints
 
-Constraints control where apps run to allow optimizing for fault tolerance or locality. They are made up of three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
+Constraints control where apps run to enable you to optimize for fault tolerance or location. Constraints have three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
```
Comment (sascala) [inline docs/docs/constraints.md]: What do we mean by "optimize for location?"

File: docs/docs/constraints.md [hunk 2]
```diff
@@ -4,31 +4,31 @@ title: Constraints
...
 
 ## Fields
 
 ### Hostname field
 
-`hostname` field matches the slave hostnames, see `UNIQUE operator` for usage example.
+The `hostname` field matches the slave hostnames. See `UNIQUE operator`, below, for a usage example.
```
Comment (sascala) [inline docs/docs/constraints.md]: Is "hostnames" supposed to be plural here? Or does the field match only one slave hostname?

File: docs/docs/constraints.md [hunk 3]
```diff
@@ -4,31 +4,31 @@ title: Constraints
...
 
-`hostname` field supports all operators of Marathon.
+The `hostname` field supports all Marathon operators.
 
 ### Attribute field
 
-If the field name is none of the above, it will be treated as a Mesos slave attribute. Mesos slave attribute is a way to tag a slave node, see `mesos-slave --help` to learn how to set the attributes.
+If the field name is not the slave hostname, it will be treated as a Mesos slave attribute. A Mesos slave attribute tags a slave node, see `mesos-slave --help` to learn how to set the attributes.
```
Comment (sascala) [inline docs/docs/constraints.md]: What do we mean by "tags"?

File: docs/docs/constraints.md (hunk 1 above)
Comment (aquamatthias) [inline docs/docs/constraints.md]: optimize for fault tolerance: try to spread tasks of an app to as many different hosts as possible.
optimize for location: try to pack as many tasks of an app to the same node.

File: docs/docs/constraints.md (hunk 2 above)
Comment (aquamatthias) [inline docs/docs/constraints.md]: We try to match offers. Every offer is from one agent on one host. No plural.

File: docs/docs/constraints.md (hunk 3 above)
Comment (aquamatthias) [inline docs/docs/constraints.md]: You can attach key/value pairs (attributes) to slaves. You can match on those attributes.

**Comment by aquamatthias** [comment]
Close for now - please reopen when ready.

**Pull request closed without merging**